xla/stream_executor/gpu/gpu_stream.h

```diff
@@ -81,6 +83,7 @@ class GpuStream : public internal::StreamInterface {
   GpuExecutor* parent_;         // Executor that spawned this stream.
   GpuStreamHandle gpu_stream_;  // Wrapped CUDA stream handle.
   int priority_ = 0;
```

Comment:
can we remove the int priority and its accessor? I think they will be unused after this change.

Reply:
Yeah, I was thinking you can do the StreamPriority -> int priority translation just for calling GPUDriver::CreateStream() but don't need to keep it around as a data member in the class

Reply:
done. GPUDriver has a function to return priority integer based on input StreamPriority.